unity_buy_android/checkstyle/config.xml

```diff
@@ -0,0 +1,223 @@
+
+        <module name="AtclauseOrder">
+            <property name="tagOrder" value="@param, @return, @throws, @deprecated"/>
+            <property name="target"
+                      value="CLASS_DEF, INTERFACE_DEF, ENUM_DEF, METHOD_DEF, CTOR_DEF, VARIABLE_DEF"/>
+        </module>
+        <module name="MethodName">
+            <property name="format" value="^[a-z][a-z0-9][a-zA-Z0-9_]*$"/>
+        </module>
+        <module name="CommentsIndentation"/>
+        <module name="IllegalImport"/>
+        <module name="RedundantImport"/>
+        <module name="UnusedImports">
+            <property name="processJavadoc" value="true"/>
+        </module>
+        <module name="ParameterNumber">
+            <property name="ignoreOverriddenMethods" value="true"/>
+        </module>
+        <module name="EmptyForIteratorPad"/>
+        <module name="NoWhitespaceAfter"/>
+        <module name="NoWhitespaceBefore"/>
+        <module name="ParenPad"/>
+        <module name="TypecastParenPad"/>
+        <module name="WhitespaceAfter"/>
+        <module name="RedundantModifier"/>
+        <module name="AvoidNestedBlocks"/>
+        <module name="CovariantEquals"/>
+        <module name="EmptyStatement"/>
+        <module name="IllegalInstantiation"/>
+        <module name="InnerAssignment"/>
+        <module name="SimplifyBooleanExpression"/>
+        <module name="SimplifyBooleanReturn"/>
+        <module name="InterfaceIsType"/>
+
+        <module name="RegexpSinglelineJava">
+            <property name="format" value="new SimpleDateFormat"/>
+            <property name="ignoreCase" value="false"/>
+            <property name="message" value="Allocating SimpleDateFormat directly can break UI tests. Use DateFormatFactory.createSimpleDateFormat() instead."/>
+        </module>
+
+        <module name="RegexpSinglelineJava">
+            <property name="format" value="Calendar.getInstance"/>
+            <property name="ignoreCase" value="false"/>
+            <property name="message" value="Using Calendar.getInstance() directly can break UI tests. Use SimpleRelativeDateUtil.now() instead."/>
+        </module>
+
+        <!-- Errors -->
+        <!--
+        <module name="Indentation"/>
+
+        <module name="OverloadMethodsDeclarationOrder"/>
+        <module name="VariableDeclarationUsageDistance"/>
+            <module name="OperatorWrap">
+                <property name="option" value="NL"/>
+                <property name="tokens" value="BAND, BOR, BSR, BXOR, DIV, EQUAL, GE, GT, LAND, LE, LITERAL_INSTANCEOF, LOR, LT, MINUS, MOD, NOT_EQUAL, PLUS, QUESTION, SL, SR, STAR "/>
+            </module>
+                    <module name="EmptyCatchBlock">
+                <property name="exceptionVariableName" value="expected"/>
+            </module>
+
+                <module name="LocalFinalVariableName"/>
+        <module name="StaticVariableName"/>
+        <module name="ConstantName"/>
+        -->
```

Comment:
There's a lot of commented out lines here, will we need these lines? If not should they be removed?